app/scripts/views/mixins/coppa-mixin.js

```diff
@@ -43,7 +43,7 @@ define(function (require, exports, module) {
       },
 
       setInitialContext (context) {
-        const coppaHTML = this.renderTemplate(Template);
+        const coppaHTML = this.renderTemplate(Template, { required: config.required });
```

Comment:
I missed this in the original coppa extraction PR. Added a test for the missing case too.